pkg/reduce/readloop/readloop.go

```diff
@@ -50,6 +51,8 @@ import (
 
 // ReadLoop is responsible for reading and forwarding the message from ISB to PBQ.
 type ReadLoop struct {
```

Comment:
we also need to capture the metric purely around udf invocation - after the close of book until a result shows up.